app/assets/stylesheets/application.scss

```diff
@@ -19,6 +19,14 @@
 
  @import "bootstrap-sass-official/assets/stylesheets/bootstrap-sprockets";
  @import "bootstrap-sass-official/assets/stylesheets/bootstrap";
+
+.top-space-40{
```

Comment:
it would be best to use this file only for importing/including bootstrap/fonts/lib etc so let's move this styles to common.scss :)